google/oauth2/id_token.py

```diff
@@ -33,9 +33,6 @@
     id_info = id_token.verify_oauth2_token(
         token, request, 'my-client-id.example.com')
 
-    if id_info['iss'] != 'https://accounts.google.com':
```

Comment:
This change does make it less clear that an exception is raised (from `verify_oauth2_token`)

Reply:
LGTM.  We don't normally track exceptions raised by other functions, anyway.